docs/DeveloperGuide.md

```diff
@@ -1125,34 +1125,73 @@ testers are expected to do more *exploratory* testing.
    
 ### Removing a contact's field
 
-1. _{ more test cases …​ }_
+1. Removing a contact's field while all contacts are being shown
+    1. Prerequisites: At least one contact in the shown list.
+
+    2. Test case: `remove 1 l/`<br>
+       Expected: First contact language field is removed. Language field in the person detail panel becomes empty.
```

Comment:
Perhaps "The language field of the first contact" or "The first contact's language field" instead; same for the following cases